projects/cps-ui-kit/src/lib/components/cps-file-upload/cps-file-upload.component.ts

```diff
@@ -77,6 +79,8 @@ export class CpsFileUploadComponent implements OnInit, OnChanges {
    */
   @Output() uploadedFileRemoved = new EventEmitter<string>();
 
+  @ViewChild('fileInput') fileInput!: ElementRef<HTMLInputElement>;
```

Comment:
```suggestion
  @ViewChild('fileInput') fileInput?: ElementRef<HTMLInputElement>;
```
I'd suggest using optional property instead of non-null assertion